addons/im_livechat/static/tests/website_livechat_window_tests.js

```diff
@@ -1,10 +1,9 @@
-odoo.define('mail.websiteLivechatWindowTests', function (require) {
+odoo.define('mail.websitelivechatwindow_test', function (require) {
```

Comment:
No, prefix should be `im_livechat` instead of `mail`, but the rest should stay the same (camelCase)